eng/pipelines/build-notice.yml

```diff
@@ -78,6 +78,36 @@ jobs:
 
       - task: ComponentGovernanceComponentDetection@0
 
+      - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
+        displayName: 'Run APIScan'
+        condition: and(succeeded(), eq(variables['OSVmImage'], 'windows'))
+        inputs:
+          softwareFolder: $(Build.ArtifactStagingDirectory)
+          # SoftwareName cannot change as it is used for looking up baselining data
+          softwareName: 'Azure Developer CLI'
+          softwareVersionNum: '$(CLI_VERSION)'
```

Comment:
This version number is the "expected next release" version number... Should this include other unique identifiers for the build?

Reply:
I don't think they are particular for this version number.